src/tests/jobtest.c

```diff
@@ -221,6 +239,18 @@ main(int argc, char **argv)
 				       " %" PRIu64 " rc: %d\n",
 				       pool_handles[i][j].cookie, rc);
 			}
+
+			uuid_unparse(c_uuid, cstr);
+			printf("Opening container %s\n",cstr);
+			rc = daos_cont_open(pool_handles[i][j], c_uuid, DAOS_COO_RW,
+						&cont_handles[i][j], NULL, NULL);
+			if (rc != 0) {
+				char		uuid_str[64];
```

Comment:
I think we have a DAOS UUID length constant declared somewhere.

Reply:
Fixed. I'm not sure why I chose 64 either since that's not the size.